.github/workflows/pluto-workflow-kustomize.yaml

```
@@ -28,5 +40,16 @@ jobs:
           UNENCRYPTED_TARGET_VERSION: ${{ secrets.TARGET_K8S_VERSION }}
           UNENCRYPTED_OPSLEVEL_PLUTO_INTEGRATION_WEBHOOK_URL: ${{ secrets.OPSLEVEL_PLUTO_INTEGRATION_WEBHOOK_URL }}
         run: |
-          kubectl kustomize ${{ inputs.template_dir }} | pluto detect - --target-versions k8s=$UNENCRYPTED_TARGET_VERSION --ignore-deprecations --ignore-removals --output json | curl -i -s -X POST $UNENCRYPTED_OPSLEVEL_PLUTO_INTEGRATION_WEBHOOK_URL?alias=${{ inputs.service_name }} -H 'content-type: application/json' --data-binary @-
+          MANIFEST_FILE=manifest.yaml
+
+          # Generate a manifest file
+          ## Kustomize
+          if [[ -n "${{ inputs.template_dir }}" ]]; then
+            kubectl kustomize ${{ inputs.template_dir }} --output=$MANIFEST_FILE
+
+          ## Helm
+          else
+            helm template charts/${{ inputs.service_name }} --values ${{ inputs.values_base }} --values ${{ inputs.values_overlay }} > $MANIFEST_FILE
+          fi
```

In shell, you can never be too safe.  Consider properly quoting arguments/redirect targets (the shell will perform word splitting on arguments after doing variable replacement).

```suggestion
          if [[ -n "${{ inputs.template_dir }}" ]]; then
            kubectl kustomize "${{ inputs.template_dir }}" "--output=$MANIFEST_FILE"

          ## Helm
          else
            helm template "charts/${{ inputs.service_name }}" --values "${{ inputs.values_base }}" --values "${{ inputs.values_overlay }}" > "$MANIFEST_FILE"
          fi
```

---

made a small tweak from your suggestion to pass the test